### ISSUE TITLE
unknown status

### ISSUE THREAD

**Issue by F3rr3ir4** [Denvi/Candle#138]
Good Morning.

I am compiled in Qt 5.4.2 (MSVC 2013, 32 bit) and it works everything except the status that is always locked in unknown after the connection. What can I do to correct this?

Thanks.

**Comment by Denvi** [comment]
Hi, look like you using new GRBL firmware and source code of Candle in master branch. Try to checkout 'grbl_1_1' branch.

**Comment by F3rr3ir4** [comment]
Yes I am using GRBL v1.1, if I use the version of Candle already compiled the status works. But if I compile the code so the status does not work, I think I've seen somewhere that the MSVC 2013 method is different.
